setup.py

```diff
@@ -12,7 +12,7 @@
         description='Qubes OS Manager',
         license='GPL2+',
         url='https://www.qubes-os.org/',
-        packages=setuptools.find_packages(),
+        packages=setuptools.find_packages(exclude=['qubesmanager.tests']),
```

Comment:
Why? Since tests needs to be run on actual system, better include them in the package. If having them installed by default is such a bad idea (?), then put them in separate sub-package (`%package` section in spec file).